### ISSUE TITLE
Timeline search

### ISSUE THREAD

**Issue by DS-INRA** [IQSS/dataverse-frontend#278]
### Overview of the Feature Request
Akin to a map search, users from History or archeology domains asked us to be able to search for dataverse using a timeline to define upper and lower dates to filter datasets based on timePeriodCovered field.

### What kind of user is the feature intended for?
Guests, UI users

Note : maybe this would not be interesting for the majority of installation and should be moved to a plugin

**Comment by pdurbin** [comment]
I'd say there has been significant interest in some kind of timeline search or date slider. Various old issues:

- https://github.com/IQSS/dataverse/issues/70
- https://github.com/IQSS/dataverse/issues/4977
- https://github.com/IQSS/dataverse/issues/8813

Here's a screenshot from 8813 by @adam3smith:

<img width="567" alt="Screenshot 2024-01-09 at 5 12 20 PM" src="https://github.com/IQSS/dataverse-frontend/assets/21006/4d41303c-f4de-4f7a-8235-50b4a589608e">

**Comment by adam3smith** [comment]
Right, I think this is universally desirable for date of publication but once that's implemented, the same facet GUI could be used for other dates

**Comment by pdurbin** [comment]
Please see this new PR:

- https://github.com/IQSS/dataverse/pull/10887
